yarn.lock

```diff
@@ -2414,7 +2414,12 @@
   dependencies:
     "@contentful/rich-text-types" "^15.12.1"
 
-"@contentful/rich-text-types@15.14.0", "@contentful/rich-text-types@^15.12.1":
+"@contentful/rich-text-types@15.14.1":
+  version "15.14.1"
+  resolved "https://registry.yarnpkg.com/@contentful/rich-text-types/-/rich-text-types-15.14.1.tgz#292f4560d67165af0ced1546a22ae77df96325ff"
+  integrity sha512-1sPVDz7MBuCCMWGvfZXQ06SqBTtlLN5GCP5ddU4c5tfatxVXhv2h80/pDCeTZVV2oCMw02YuNmBgql60+0oL4Q==
+
+"@contentful/rich-text-types@^15.12.1":
```

Comment:
Looks like we still got to bump rt types version in the renderer so this can be deduped.